### PR TITLE
Add basic k8s/k9s info

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,92 @@
...
 ## Go
-For those that cherish the classroom experience Udemy has a great course that covers a range from fundamentals all the way to advanced language features. This is probably the best place to start for most people.  
-- [Learn how to code](https://www.udemy.com/course/learn-how-to-code/)
 
-For people that prefer the rustle of paper and self directed learning then the best place to start is *the* Go book
-- [The Go Programming Language](http://www.gopl.io/)
+For those that cherish the classroom experience Udemy has a great course that covers a range from fundamentals all the way to advanced language features.
+
+-   [Learn How To Code](https://www.udemy.com/course/learn-how-to-code/)
+
```
Comment (cdempsie) [inline README.md:85]: Tour of Go is also good and quick to get through:
https://go.dev/tour/welcome/1

File: README.md [hunk 2]
```diff
@@ -1,31 +1,92 @@
...
+For people that prefer the rustle of paper and self directed learning then the best place to start is _the_ Go book
+
+-   [The Go Programming Language](http://www.gopl.io/)
```
Comment (cdempsie) [inline README.md:88]: There are a couple of newer Go books which were out last year which I found less academic than the original book:
- [Learning Go](https://www.oreilly.com/library/view/learning-go/9781492077206/)
- [Cloud Native Go](https://www.oreilly.com/library/view/cloud-native-go/9781492076322/)

Comment (ghost) [reply README.md:88]: Thanks for these, I could well be wrong but I feel the person that wants to read a book over a website / course is the person that doesn't mind the academic approach and I'd like to keep the options low so its not overwhelming.  Maybe we can take a poll and see what's the one book everyone would recommend and had that be the default.

Comment (rbulloch) [reply README.md:88]: They'd found in my last business that alongside the standard Go book ([The Go Programming Language](http://www.gopl.io/) a really good supplement to this was 'Concurrency in Go: Tools and Techniques for Developers
by Katherine Cox-Buday' for the teams involved in large amounts of frequent transactions/events.

**Pull request merged**